solarextrapolation/utilities.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jul 27 15:32:51 2015
+
+@author: alex_
+"""
+
+from astropy import units as u
+import numpy as np
+import sunpy.map as mp
+from copy import deepcopy
+
+def decompose_ang_len(qua_input, **kwargs):
+    """
+    Function to help decompose quantities that have an equivilence between angles
+    and length, such as photospheric observational angles and object sizes.
+    The function uses an equivilence to cobnvert into eithe length or angle
+    physical_type units.
+
+    Parameters
+    ----------
+    
```

Comment:
Parameters should also include a description of the arguments, not just kwargs. Also unless you have a reason for doing `**kwargs` you should just specify them in the function signature, as it makes it easier to inspect.